completions/okular

```diff
@@ -0,0 +1 @@
+complete -W '-p --page --presentation --print --print-and-exit --unique --noraise --find --editor-cmd' okular
```

Comment:
This setting overwrites the existing okular completion by `bash-completion`. `bash-completion` already provides [the completion for `okular` by `_filedir_xspec`](https://github.com/scop/bash-completion/blob/79e9d9392f4f9dbb65177d9554321df5d5b064c5/bash_completion#L2694-L2695), which provides basic completions such as variable name completion, filenames of redirections, [`xspec` for okular](https://github.com/scop/bash-completion/blob/79e9d9392f4f9dbb65177d9554321df5d5b064c5/bash_completion#L2557), etc. These features will be lost by this PR, though it adds a completion for options.